Packages/com.unity.inputsystem/package.json

```diff
@@ -3,12 +3,7 @@
 	"displayName": "Input System",
 	"version": "1.1.0-preview.2",
 	"unity": "2019.4",
-	"repository": {
-		"type": "git",
-		"url": "https://github.com/Unity-Technologies/InputSystem.git",
-		"revision": "9ceb1563f669e33fe7b8562e36375caa59bb6f1b"
```

Comment:
It's the entire section that is added by release tooling and not just the revision?

Reply:
Yes, whole section